src/lib/routes/admin-api/tag-type.ts

```diff
@@ -180,10 +180,16 @@ class TagTypeController extends Controller {
         res: Response<ValidateTagTypeSchema>,
     ): Promise<void> {
         await this.tagTypeService.validate(req.body);
-        this.openApiService.respondWithValidation(200, res, tagTypeSchema.$id, {
-            valid: true,
-            tagType: req.body,
-        });
+
+        this.openApiService.respondWithValidation(
+            200,
+            res,
+            validateTagTypeSchema.$id,
+            {
+                valid: true,
+                tagType: req.body,
+            },
+        );
```

Comment:
The only thing that's changed here (aside from formatting) is that we use the correct `validateTagTypeSchema.$id` instead of `tagTypeSchema.$id` in the response. This was probably a typo or copy/paste error.